core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -1284,16 +1284,26 @@ private List<KafkaConnectorRequestsResponseModel> getConnectorRequestModels(
       if (RequestStatus.APPROVED != kafkaConnectorRequestModel.getRequestStatus()) {
         if (kafkaConnectorRequestModel.getRequestOperationType() != null
             && RequestOperationType.CLAIM == kafkaConnectorRequestModel.getRequestOperationType()) {
+
           List<KwKafkaConnector> topics =
               getConnectorsFromName(kafkaConnectorRequestModel.getConnectorName(), tenantId);
-          kafkaConnectorRequestModel.setApprovingTeamDetails(
-              updateApproverInfo(
-                  manageDatabase
-                      .getHandleDbRequests()
-                      .getAllUsersInfoForTeam(topics.get(0).getTeamId(), tenantId),
-                  manageDatabase.getTeamNameFromTeamId(tenantId, topics.get(0).getTeamId()),
-                  approverRoles,
-                  kafkaConnectorRequestModel.getRequestor()));
+          if (!topics.isEmpty()) {
```

Comment:
```suggestion
          if (!connectors.isEmpty()) {
```
may be we can rename this variable